packer/install-dependencies.sh

```diff
@@ -331,21 +361,27 @@ IP_MASQ_AGENT_VERSIONS="
 2.0.0
 "
 for IP_MASQ_AGENT_VERSION in ${IP_MASQ_AGENT_VERSIONS}; do
-    # TODO remove the gcr.io/google-containers image once AKS switches to use k8s.gcr.io
-    DEPRECATED_CONTAINER_IMAGE="gcr.io/google-containers/ip-masq-agent-amd64:v${IP_MASQ_AGENT_VERSION}"
-    pullContainerImage "docker" ${DEPRECATED_CONTAINER_IMAGE}
-    echo "  - ${DEPRECATED_CONTAINER_IMAGE}" >> ${RELEASE_NOTES_FILEPATH}
+  # TODO remove the gcr.io/google-containers image once AKS switches to use k8s.gcr.io
+  DEPRECATED_CONTAINER_IMAGE="gcr.io/google-containers/ip-masq-agent-amd64:v${IP_MASQ_AGENT_VERSION}"
+  pullContainerImage "docker" ${DEPRECATED_CONTAINER_IMAGE}
+  echo "  - ${DEPRECATED_CONTAINER_IMAGE}" >> ${RELEASE_NOTES_FILEPATH}
 
-    CONTAINER_IMAGE="k8s.gcr.io/ip-masq-agent-amd64:v${IP_MASQ_AGENT_VERSION}"
+  for IMAGE_BASE in k8s.gcr.io aksrepos.azurecr.io/mirror; do
+    CONTAINER_IMAGE="${IMAGE_BASE}/ip-masq-agent-amd64:v${IP_MASQ_AGENT_VERSION}"
     pullContainerImage "docker" ${CONTAINER_IMAGE}
     echo "  - ${CONTAINER_IMAGE}" >> ${RELEASE_NOTES_FILEPATH}
+  done
 done
 
 NGINX_VERSIONS="1.13.12-alpine"
 for NGINX_VERSION in ${NGINX_VERSIONS}; do
     CONTAINER_IMAGE="nginx:${NGINX_VERSION}"
     pullContainerImage "docker" ${CONTAINER_IMAGE}
     echo "  - ${CONTAINER_IMAGE}" >> ${RELEASE_NOTES_FILEPATH}
+
+    CONTAINER_IMAGE="aksrepos.azurecr.io/mirror/nginx:${NGINX_VERSION}"
```

Comment:
why not do a for loop on IMAGE_BASE here?

Reply:
changed to for loop.